root/static/scripts/common/utility/isDatabaseRowId.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * @flow strict
+ * Copyright (C) 2022 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const MAX_POSTGRES_INT = 2_147_483_647;
+
+export default function isDatabaseRowId(input: mixed): boolean %checks {
```

Comment:
Is this going to be removed from Perl eventually? If not, maybe link the two with a comment?

Reply:
I don't think it will be removed from Perl, but I added some comments.